### ISSUE TITLE
df.to_gbq(): InvalidSchema when using 'type': 'INT64' and if_exists = 'append'

### ISSUE THREAD

**Issue by Azerane** [pydata/pandas-gbq#322]
Hello there,

I'm facing an issue with the function `df.to_gbq()`

From my understanding, it seems that when we are using `if_exists = 'append'` when the destination_table already exists, we are retrieving the original_schema from destination_table.

The original_schema will contain every fields but type retrieve is `INTEGER` and not `INT64` anymore.
Since the code is simply checking that all fields in the `schema_local` are the same in the `schema_remote`, this conducts to a missmatch and function `schema_is_subset` is returning `False` instead of `True`.

I'm not sure about where we should fix this error, but it's kind of annoying when working with integers values.

**Comment by norbertsteele** [comment]
Is there a workaround that anyone has found?

**Comment by gilangardya** [comment]
Hi @Azerane,

Maybe you get that error because you use `object` instead of `int64` to represent some integer values.
If you need `object` representation because you store big-integer (i.e. 2**65), you should change the data type since BigQuery `INTEGER` range is -2^63 .. 2^63-1.
But if you don't need that, you can cast the Series into `int64` first before loading into the BigQuery table, or you can specify the `table_schema` in `to_gbq` parameters.
If that's not your case, please share more details, that would be great.

Maybe it can help you too @norbertsteele

**Comment by tswast** [comment]
I'm able to reproduce with the following code example:

```python
import pandas_gbq

df1 = pandas.DataFrame({"int_col": [1, 2, 3]})
pandas_gbq.to_gbq(
    df1,
    "my_dataset.int64table",
    table_schema=[{"name": "int_col", "type": "INT64"}]
)
df2 = pandas.DataFrame({"int_col": [4, 5, 6]})

# This fails, despite using the same schema as was used for the initial
# upload.
pandas_gbq.to_gbq(
    df2,
    "my_dataset.int64table",
    table_schema=[{"name": "int_col", "type": "INT64"}],
    if_exists="append",
```

**Comment by tswast** [comment]
> Is there a workaround that anyone has found?

Workaround is to use legacy SQL type names (`INTEGER`, not `INT64`), but this should be fixed once https://github.com/pydata/pandas-gbq/pull/340 is merged and released (target: 0.14.1)

**Comment by SDonatas** [comment]
Had this error in few days. The problem is with wrong error handling. The issue I had was - pandas dataframe did not have datetime columns converted as specified in supplied schema. The integer error, at least for me was a false flag.
